src/app/components/TradeTable/index.jsx

```diff
@@ -1,108 +1,16 @@
 import React, { Fragment } from 'react'
-import { push as pushAction } from 'react-router-redux'
-import { connect } from 'react-redux'
-import { Table } from 'reactstrap'
-import { compose, setDisplayName, setPropTypes, defaultProps, withHandlers } from 'recompose'
-import PropTypes from 'prop-types'
-import classNames from 'class-names'
+import { compose, setDisplayName } from 'recompose'
+import CompletedOrdersTable from 'Components/TradeTable/CompletedOrdersTable'
+import PendingOrdersTable from 'Components/TradeTable/PendingOrdersTable'
 
-import routes from 'Routes'
-import Units from 'Components/Units'
-import Expandable from 'Components/Expandable'
-import CoinIcon from 'Components/CoinIcon'
 
-import { tradeTable, tradeCoinIcon } from './style'
-
-const NODATA = '-----'
-
-const CoinSymbol = ({ symbol, ...props }) => (
+const TradeTable = () => (
   <Fragment>
-    <CoinIcon className={classNames(tradeCoinIcon, 'mr-1')} symbol={symbol} size='sm' inline {...props}/>
-    {symbol}
+    <PendingOrdersTable/>
```

Comment:
You've duplicated a ton of logic by splitting up your trade table. Can you refactor this so that there's only one generic TradeTable that used by both CompletedOrdersTable and PendingOrdersTable and move them both into the TradeHistory container.